src/test/resources/mappings/getBooking_A7090BF.json

```diff
@@ -123,21 +123,6 @@
         "paroleEligibilityDate": "2020-02-03",
         "actualParoleDate": "2020-02-03"
       },
-      "restrictivePatient": {
```

Comment:
This has been removed from the prison api - replaced with a call to the Restricted Patient API